### PR TITLE
Feature 1/setup basic repo configuration

### PR DESCRIPTION
**Pull request by Sofinaye**
In this feature I have added:

- Html code 

- CSS code 

- stylelintrc.json file

- .gitignore file

- Readme file

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @Sofinaye , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

__Kindly make sure to use the same PR and repo for re-review as it allows us to keep track of your evolution__

## Highlights :tophat:  :+1: 
- Good use of GitHub flow :+1: 
- test.md and node_modules are ignored :+1: 
## Required Changes ♻️, 
- As per the project requirements kindly set up linters by visiting the following [repo](https://github.com/microverseinc/linters-config).
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+
+> Describe between 1-3 key features of the application.
+
+- **[Git-ignore]**
+- **[Html-header]**
+- **[]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
```
Comment (GedeonTS) [inline README.md]: - Kindly make sure to have all the missing sections of the README template considering the FAQ as the only optional section. :recycle: please remember to customize them too

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Sofinaye,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Readme file professional ✔️ 
- Commit message ✔️

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] As suggested by the previous reviewer, the project requirements require setting up linters. You can get it done by following [this repo steps](https://github.com/microverseinc/linters-config). Linters checking is a crucial part of developers as it helps you to follow the industrial standard and guides you in writing professionally standard code.
- [ ] Kindly resolve the path of the issue on your readme file, it is currently goes to 404 page.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Eternalgratis — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
Hi @Sofinaye , 

In my opinion, your project is ready for submission!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 
Developer-512px-10


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**